### ISSUE TITLE
Tray icons context menu seems broken on 1st click

### ISSUE THREAD

**Issue by Jazzhoo** [vinceliuice/WhiteSur-gtk-theme#158]
Hi, 
I'm not sure if that's the theme issue, but here is it. 

When I click on the icons in my tray (notification) area the context menu size and look is often broken, the second click fixes the issue. It sometimes affects the parent context window and mostly the child (invoked by arrow sub-item). 

Issue gets fixed also by taking the focus out of the affected arrow item, and focusing back.

Also, the style of the parent context window is squared without nice rounded corners like the child context. 

Additional info, when the application e.g. Skype or Teams is terminated and re-opened the issue occurs again. 

I'm using cinnamon 4.8.6. 
Photos attached. 
Microsoft Teams: 

![Initial_context_menu](https://user-images.githubusercontent.com/53535260/111661451-efd7a300-8806-11eb-9c99-0922e02ca161.jpg)
![refreshed_context_menu](https://user-images.githubusercontent.com/53535260/111661455-f108d000-8806-11eb-84a8-da6d5a9a5084.jpg)

Additional from Skype: 

![broken](https://user-images.githubusercontent.com/53535260/111662438-d3883600-8807-11eb-932c-56c8300f16e6.jpg)
![fixed](https://user-images.githubusercontent.com/53535260/111662496-e00c8e80-8807-11eb-848e-2ce94418c954.jpg)

Other than that I think the theme is great!! Good work and thank you!!

**Comment by rivanfebrian123** [comment]
Hi Jan! I think it's GTK issue (or Cinnamon issue). Try to open your issue here: https://gitlab.gnome.org/GNOME/gtk/-/issues. Anyway, does that thing also happen on other themes?

**Comment by Jazzhoo** [comment]
Tested with Mint-Y, Flat-Remix-GTK and Manjaro's native Matcha. No issue with those themes.

**Comment by rivanfebrian123** [comment]
Hey, you know what? Matcha is Vince's theme! @vinceliuice I think we need to re-derivate our theme from your Matcha theme, make em consistent to each other. It'll probably take 4 months to finish. Just check this out for comparison: https://github.com/rivanfebrian123/WhiteSur-gtk-theme **vs** 
[WhiteSur-gtk-theme-tested.zip](https://github.com/vinceliuice/WhiteSur-gtk-theme/files/6183055/WhiteSur-gtk-theme-tested.zip)

**Comment by Jazzhoo** [comment]
I've tried the "tested" one. Unfortunately the same issue.

I'm wondering why that issue hasn't been reported previously by anyone? Am I the only one experiencing it? - that would be weird.
I remember, I have already had similar issue with another MAC-like theme on Gnome some time ago (can't remember the name exactly.. , maybe it was WhiteSur's ancestor?)

**Comment by rivanfebrian123** [comment]
> I've tried the "tested" one. Unfortunately the same issue.
Yeah, I admit that. It need 3 more weeks to finish since my last commit here: https://github.com/vinceliuice/WhiteSur-gtk-theme/commits/master. But hey! At least you can try some features in ./install -h and ./tweaks.sh, in the mean time. Customized theme installation (`-c`, `-p`, `-t`, etc.) speed is also improved by around 3 times!

> I'll report after a day or two of testing
Cool!

> maybe it was WhiteSur's ancestor?
Probably yes, Vince created lots of theme including ones here: https://www.omgubuntu.co.uk/2017/11/best-gtk-themes-for-ubuntu . I think he's kinda crazy :D

**Comment by Jazzhoo** [comment]
The issue doesn't exist with Cinnamon's standard tray applet.

**Comment by Jazzhoo** [comment]
Is it possible to add support for Cinnamon's XApp Status Applet. It is officially used by Linux Mint to support tray area.
